Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -43,6 +43,8 @@ class XMLHttpRequestBase {
   readyState: number;
   responseHeaders: ?Object;
   responseText: ?string;
+  response: ?Object;
+  responseType: ?string;
```

Comment:
Yeah, but we already know the values, so we can be more specific. The following covers everything we support right now,
```js
responseType: '' | 'text' | 'json';
```